legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/GitLabProjectApi.java

```diff
@@ -996,10 +1005,19 @@ public List<String> getTags()
         }
 
         @Override
-        @JsonIgnore
         public ProjectType getProjectType()
         {
-            return null;
+            ProjectConfiguration projectConfiguration = getProjectConfiguration(projectId);
+            ProjectType projectType = null;
+            if (projectConfiguration != null)
+            {
+                projectType = projectConfiguration.getProjectType();
+            }
+            if (!ProjectStructure.isValidProjectType(projectType))
+            {
+                projectType = ProjectType.MANAGED;
+            }
+            return projectType;
```

Comment:
We cannot do file access here. This should still return null, and we should probably leave @JsonIgnore.

Reply:
done